cfme/fixtures/nuage.py

```diff
@@ -58,3 +58,26 @@ def with_nuage_sandbox(networks_provider):
     enterprise = sandbox['enterprise']
     nuage.delete_enterprise(enterprise)
     logger.info('Destroyed sandbox enterprise {} ({})'.format(enterprise.name, enterprise.id))
+
+
+@pytest.fixture(scope='module')
+def with_nuage_sandbox_modscope(appliance, setup_provider_modscope, provider):
+    nuage = provider.mgmt
+    sandbox = create_basic_sandbox(nuage)
+    enterprise = sandbox['enterprise']
+    logger.info('Performing a full refresh, so sandbox %s appears in the database', enterprise.name)
+    wait_for(provider.is_refreshed, func_kwargs=dict(refresh_delta=10), timeout=600, delay=10)
+
+    # Check if tenant exists in database, if not fail test immediately
+    tenants_table = appliance.db.client['cloud_tenants']
+    tenant = (appliance.db.client.session
+              .query(tenants_table.name, tenants_table.ems_ref)
+              .filter(tenants_table.ems_ref == enterprise.id).first())
+    assert tenant is not None, 'Nuage sandbox tenant inventory missing: {}'.format(enterprise.name)
+
+    # Let integration test do whatever it needs to do.
+    yield sandbox
+
+    # Destroy the sandbox.
+    nuage.delete_enterprise(enterprise)
+    logger.info('Destroyed sandbox enterprise {} ({})'.format(enterprise.name, enterprise.id))
```

Comment:
I would rather create new PR for this and fix all string formatting in this file, because those lines already existed before I started implementing new test. I didn't want to change things that weren't related.